src/analyzer/plugins/analyzerkeyfinder.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+#include <keyfinder/keyfinder.h>
+
+#include <QObject>
+
+#include "analyzer/plugins/analyzerplugin.h"
+#include "util/types.h"
+
+namespace mixxx {
+
+class AnalyzerKeyFinder : public AnalyzerKeyPlugin {
+  public:
+    static AnalyzerPluginInfo pluginInfo() {
+        return AnalyzerPluginInfo(
+                "keyfinder:2",
```

Comment:
The `foo:index` plugin ID is a VAMP-ism that we don't have to follow if we don't want to. I don't think anything in Mixxx relies on it any more, so its id could just be `keyfinder`

Reply:
(it's still there for the existing plugins because it gets written to user's databases so once we pick one it needs to stay constant)

Reply:
I used `:2` for the library version.

Reply:
Ok, could you document that? I think it's confusing to use colon because it's so similar to VAMP's plugin spec. Maybe hyphen would be clearer.